### PR TITLE
Dan freeman was really bothered by this

### PR DESCRIPTION
**Pull request by brimatteng**
This was just a small error for formating in certain ruby versions. It should just be a hash

prime @brea9482 
cc @dfreeman (I fixed this for you)

### CODE REVIEW

File: spec/arc-furnace/all_fields_csv_sink_spec.rb [hunk 1]
```diff
@@ -22,8 +22,8 @@
   describe 'With symbolized keys' do
     before do
       sink.prepare
-      sink.row(id: "111", "Field 1": "boo bar", "Field 2": "baz, bar")
-      sink.row(id: "222", "Field 1": [ "baz", "bag" ], "Field 2": "boo bar")
+      sink.row('id' => '111', 'Field 1' => 'boo bar', 'Field 2' => 'baz, bar')
+      sink.row('id' => '222', 'Field 1' => [ 'baz', 'bag' ], 'Field 2' => 'boo bar')
       sink.finalize
```
Comment (brea9482) [inline spec/arc-furnace/all_fields_csv_sink_spec.rb]: this change defeats the purpose of this test which is to verify that the sink can handle symbols along with strings.

Comment (brimatteng) [inline spec/arc-furnace/all_fields_csv_sink_spec.rb]: yes it does. Find and replaced too much

**Comment by brea9482** [comment]
Cool thanks, good to go.

**Comment by dfreeman** [comment]
🎉 🎊

**Pull request merged**